configuration/persistence.md

```diff
@@ -7,12 +7,238 @@ title: Persistence
-- [The openHAB 1.x wiki persistence article](https://github.com/openhab/openhab/wiki/Persistence)
+openHAB has the ability to save item states according to rules you have configured.
+This is called persistence.
+Persistence can have several important uses.
+For example, persistence allows an openHAB system to restore itself to the state it was in prior to a restart (if you have configured it to do so).
+Persistence may also be used to store values that are subsequently displayed in a graph.
+(For an excellent graphing tutorial see [InfluxDB+Grafana persistence and graphing](https://community.openhab.org/t/influxdb-grafana-persistence-and-graphing/13761)
+
+openHAB persists item states in a database, and most popular databases are supported.
+You may have more that one persistence add-on loaded, and each of these may be configured independently.
+Examples of available persistence add-ons include relational databases, NoSQL databases, round-robin databases, Internet-of-Things (IoT) cloud services, simple log files, etc.
+All of these options are available in openHAB, and they are all configured in the same way.
+Note that the type of database(s) you use for persistence may be influenced by your requirements for exporting data (e.g. IoT services or log files), or your requirement to make queries against the database to retrieve historical data (e.g. MySQL and MongoDB).
+A complete list of supported persistence add-ons may be found in the [persistence]({{base}}/addons/persistence.html) section of the on-line openHAB documentation.
+
+## Persistence Add-on Configuration
+
+Each persistence add-on you install will need to be configured.
+Please refer to the specific [on-line documentation]({{base}}/addons/persistence.html) for your selected persistence add-on for configuration instructions.
+
+## Default Persistence Service
+
+You may install more than one persistence add-on.
+Therefore, it is important to select a default persistence service.
+You should do this even if you have only one persistence add-on installed.
+
+To select a default persistence service, in paperUI, select Configuration and then System from the side menu.
+Scroll down to "Persistence", and select your Default Service from the drop-down list.
+Note that you must first install a persistence add-on before you make this selection.
+Be sure to save your choice once you have selected your default service.
+
+## configuration
+
+You can choose how your persistence service operates, which item states it persists, and under what conditions it persists them.
+These things and more are configured in a file named `<persistenceservice>.persist`, where "persistenceservice" is replaced by the name of your add-on (e.g. `rrd4j.persist`).
+This folder is located in `$OPENHAB_CONF/persistence`.
+The configuration defines persistence "strategies" which are very similar to [triggers]({{base}}/configuration/rules-dls.html#rule-triggers) in openHAB rules.
```

Comment:
The link throws an 404 not found error.

Reply:
ah i found the issue:
"dls" must be "dsl" 

-> [triggers]({{base}}/configuration/rules-dsl.html#rule-triggers)

Reply:
Agree - thank you and well spotted.  I will change this.